payment/views.py

```diff
@@ -155,9 +155,10 @@ def payment_terms_page(request, force):
     terms = TermsAndConditions.get_latest(name=PAYMENT_TERMS)
 
     return render_to_response('payment/terms-and-conditions.html', {
-            'terms': terms,
-            'force': force,
-            })
+        'force': force,
+        'query_params': request.GET.urlencode(),
```

Comment:
it might be that request is already available in template context :man_shrugging:

Reply:
Unfortunately no. There is a `urlencode` tag but not something to retrieve query params.
But instead to pass all the query params, I could just pass to the context the `next` param 🤷‍♂️

```python
'next': request.GET.get('next', '')
```

Then in `terms-and-conditions.html`
```html
<form action="${reverse('payment:accept-terms')}?${next|urlencode}" method="POST">
```

Reply:
:ok_hand: